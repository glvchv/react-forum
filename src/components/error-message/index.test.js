import React from 'react';
import ErrorMessage from '.';
import renderer from 'react-test-renderer';

describe('Error-message component', () => {
    it('should render the component with err message', () => {
        const tree = renderer.create(
            <ErrorMessage error='test'/>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})