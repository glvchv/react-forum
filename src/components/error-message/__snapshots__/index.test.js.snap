// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error-message component should render the component with err message 1`] = `
<p
  className="error"
>
  test
</p>
`;
